### DISCUSSION TITLE
Problemi e instabilità con Sonoff ZBDongle-E

### DISCUSSION THREAD

**Discussion by RCorsaletti** [indomus/forum#2984] [HUB: Home Assistant]
Buongiorno a tutti,
ho installato la Sonoff ZBDongle-E al posto di una Conbee 2 e devo dire che la fase di setup iniziale (seguendo la vostra pagina dedicata, per la quale vi ringrazio) è stata molto semplice. Ho migrato tutti i dispositivi già presenti su Zigbee2MQTT e per un po' il tutto ha funzionato. 

Ora rilevo delle latenze casuali dai sensori di movimento Philips HUE, alcune volte funzionano istantaneamente, altre volte c'è una latenza che può arrivare anche a 10 secondi. Guardando i log di Z2M quando si verifica la latenza i messaggi non vengono proprio inviati, appena arrivano HA esegue quello che deve fare. Ho verificato inoltre che durante il periodo di latenza tutti i sensori di movimento non funzionano, quindi non è un problema del singolo sensore ma di tutti.

Per rimediare ho provato a modificare il canale spostandolo al 25 e modificare il canale del wifi 2.4 a 1 ma non è cambiato nulla (posto che comunque non è mai stato un problema con la conbee 2)

Vi viene in mente qualcosa?

**Comment by indomus** [comment]
Buondì e... grazie per i ringraziamenti, sempre apprezzati :)
Che firmware stai usando? NCP o RCP? Che versione?

**Comment by RCorsaletti** [comment]
Ciao! ho installato il firmware RCP, per quanto riguarda la versione dal log di Zigbee2MQTT vedo:

Versione Zigbee2MQTT
[1.36.0](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.36.0) commit: [unknown](https://github.com/Koenkk/zigbee2mqtt/commit/unknown)
Tipo Coordinator
EZSP v12
Revisione Coordinator
7.3.1.0 build 0

**Reply by indomus** [reply]
Quindi stai usando anche Silicon Labs Multiprotocol?

**Reply by RCorsaletti** [reply]
si, aggiornato alla versione 2.4.4

**Reply by indomus** [reply]
Allora potrebbe essere quello. Se cerchi sul sito abbiamo dedicato delle news al tema, purtroppo c'è molta instabilità al momento usando RCP. (tu sei sicuro, eh. dalla [versione](https://github.com/darkxst/silabs-firmware-builder/tree/main/firmware_builds/zbdonglee) mi sembra più un NCP)

**Reply by RCorsaletti** [reply]
si per conferma ho connesso il dispositivo ad una vm windows e ho questo:
![image](https://github.com/indomus/forum/assets/26116122/47ef0080-84b7-4331-933b-daf35520b0f3)

**Reply by RCorsaletti** [reply]
quindi mi consigli di passare al firmware zigbee? avevo utilizzato questo in quanto ho in programma di acquistare una Nuki e il protocollo thread mi sarebbe stato utilile per connetterla, ma al momento non lo sto sfruttando in realtà

**Reply by RCorsaletti** [reply]
ho controllato l'articolo che citavi, stando a quello questa sarebbe la configurazione funzionante:
Firmware RCP: 4.3.1 (effettuare quindi un rollback alla versione precedente alla 4.4.0, qui la guida)
Silicon Labs Multiprotocol:
Add-on Home Assistant OS: 2.4.4
container Docker: 2.4.4
ZigBee2MQTT: 1.35.2

io al momento ho:
Firmware RCP: 4.3.1
Silicon Labs Multiprotocol Add-on Home Assistant OS: 2.4.4
ZigBee2MQTT: **1.36.0-1**


Quindi mi basterebbe fare il rollback di ZigBee2MQTT, che ne pensi?

**Reply by indomus** [reply]
ah ok 4.3.1, prima hai scritto 7.3.1.0.
beh, sicuramente con NCP last relese dovresti avere più stabilità. Se non hai immediatamente bisogno di usare Thread, puoi serenamente usare quello e temporaneamente accantonare Silicon Labs Multiprotocol.

**Reply by RCorsaletti** [reply]
si il 7.3.1 me lo dice ZigBee2MQTT quindi penso sia la versione di zigbee dentro il firmware RCP 4.3.1
